web/models/github_app_installation.ex

```diff
@@ -21,47 +24,31 @@ defmodule CodeCorps.GithubAppInstallation do
   end
 
   @doc """
-  Builds a changeset based on the `struct` and `params`.
+  Changeset used to create a GithubAppInstallation record from CodeCorps
   """
   def create_changeset(struct, params \\ %{}) do
     struct
-    |> changeset(params)
+    |> cast(params, [:project_id, :user_id])
+    |> put_change(:state, "unprocessed")
+    |> put_change(:origin, "codecorps")
+    |> validate_required([:project_id, :user_id])
+    |> assoc_constraint(:project)
+    |> assoc_constraint(:user)
   end
 
+  @doc """
+  Changeset used to update a GithubAppInstallation record from CodeCorps
+  """
   def update_changeset(struct, params \\ %{}) do
     struct
-    |> changeset(params)
+    |> cast(params, [:project_id])
+    |> validate_required([:project_id])
+    |> assoc_constraint(:project)
```

Comment:
I'm not sure we actually need an update like this.

With the introduction of `OrganizationGithubAppInstallation`, this changes anyway.